pkg/accounting/dbx/accounting.dbx

```diff
@@ -14,39 +14,51 @@ read scalar (
   where  timestamps.name = ?
 )
 
-model aggregate (
-	key node_id
-
+model rollup (
```

Comment:
I wonder if we should use longer table names, e.g. `account_rollup`, `accounting_raw`, because we end up merging them, so having a standalone table `raw` might not be that descriptive?

Reply:
true true, i'll add that to the db merging ticket